frontend/src/app/App.test.tsx

```diff
@@ -131,6 +90,52 @@ const getWrapper = (): ReactWrapper => {
   return mount(<App {...props} />, { attachTo: mountPoint })
 }
 
+function mockEventListeners(): (type: string, event: any) => void {
```

Comment:
Docstring please (which event listeners are mocked and what is the return value)